### PR TITLE
TfWM Updates

### LINKED ISSUE #33: Beta phase indicator is centred at the top of the email (iOS/Gmail)
This doesn't happen on a webpage.

IMG_0954

### LINKED ISSUE #34: Service numbers display strangely with added padding above service number (iOS/Gmail)
Doesn't happen on browser view

IMG_0954

### LINKED ISSUE #35: View disruption on map / plan journey buttons are misaligned (iOS/Gmail/Safari)
Happens in-browser too (iOS/Safari)

IMG_0956

### LINKED ISSUE #36: Footer of email has strange alignment (iOS/Gmail)
- "You received this because..." not full-width
- Links in strange order

IMG_0956

### PR DESCRIPTION
**Pull request by houbly**
- Updated links to tfwm.org.uk
- Updated to tfwm logo text
- Added hidden preheader text to show better email preview descriptions
- fixes #33, fixes #34, fixes #35, fixes #36

### CODE REVIEW

File: src/partials/header.mjml [hunk 1]
```diff
@@ -1,23 +1,23 @@
 <mj-section background-color="#3c1053" full-width="full-width" text-align="left" padding="10px 0">
   <mj-column width="52px">
     <mj-image
-      src="/assets/wmn-icon.png"
-      alt="West Midlands Network logo"
+      src="/assets/tfwm-icon.png"
+      alt="Transport for West Midlands logo"
```
Comment (mrmjprice) [inline src/partials/header.mjml:5]: Leave alt text blank for this as screen readers won't have a great experience hearing the same thing twice

File: src/partials/header.mjml [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
       align="left"
       width="42px"
-      height="39px"
+      height="40px"
       padding-top="5px"
       padding-left="10px"
       padding-right="0px"
       href="https://www.wmca.org.uk"
```
Comment (mrmjprice) [inline src/partials/header.mjml:12]: Should link to https://www.tfwm.org.uk

File: src/partials/header.mjml [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
   </mj-column>
-  <mj-column width="71px">
+  <mj-column width="111px">
     <mj-image
-      src="/assets/wmn-text.png"
-      alt="West Midlands Network logo text"
-      height="39px"
-      width="71px"
+      src="/assets/tfwm-text.png"
+      alt="Transport for West Midlands logo text"
```
Comment (mrmjprice) [inline src/partials/header.mjml:18]: Change to "Transport for West Midlands"

**Review by mrmjprice — CHANGES_REQUESTED** [review]
Couple of changes here to help screen readers and also update a link. Thanks.

**Review by catiarodriguescosta — APPROVED** [review]
👍

**Review by daylesalmon — APPROVED** [review]

**Pull request merged**